src/sdk/lsf-sdk.js

```diff
@@ -307,6 +309,16 @@ export class LSFWrapper {
     this.setLSFTask(task, annotationID, fromHistory);
   }
 
+  adjacentTaskIds() { 
+    const filteredOrderedList = this.datamanager.store.viewsStore.dataStore.list;
+    const thisTaskInView = filteredOrderedList.findIndex((task) => task.id === this.task.id);
```

Comment:
What happens if `thisTaskInView` was to not find the task in the list? Although this shouldn't happen, it would be better to handle the case up front where `thisTaskInView` is `-1`.